src/lib/Behat/Helper/Hooks.php

```diff
@@ -15,13 +15,6 @@ class Hooks extends RawMinkContext
 {
     use KernelDictionary;
 
-    /** @BeforeScenario
```

Comment:
I've decided to use this opportunity to remove this, I don't think we need it and it causes failures from time to time.